diag_manager/fms_diag_object.F90

```diff
@@ -116,7 +119,7 @@ module fms_diag_object_mod
      procedure :: has_data_RANGE
 ! Get functions
      procedure :: get_diag_id => fms_diag_get_id
-     procedure :: get_metadata
+     procedure :: get_attributes
```

Comment:
why rename this?  seems like extra work.

Reply:
just to be consistent with the axis_obj and with the `diag_field_add_attribute` function